client/src/components/ChatItem.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import KeyboardArrowLeftIcon from '@mui/icons-material/KeyboardArrowLeft';
+import OrBackground from '../assets/OrBackground.svg';
+
+export interface ChatItemProps {
+    name: string,
+    lastMessageTiming: string,
+    unreadMessages: number,
+    isEnded: boolean,
+    chatId: number
+};
+
+export const ChatItem: React.FC<ChatItemProps> = ({
+    name, lastMessageTiming, unreadMessages, isEnded, chatId }: ChatItemProps) => {
+
+    const OnItemClick = (chatId: number) => {
```

Comment:
this function does not need to accepts chatId as an argument since you already destructed it and it is known in the functions' scope